avro2tf/src/main/scala/com/linkedin/avro2tf/helpers/Avro2TFJobHelper.scala

```diff
@@ -213,10 +213,13 @@ object Avro2TFJobHelper {
           // Construct a new array of String array column for String array data type to be used with SequenceExample
           newConvertedColumns.append(wrapStringArray(dataFrame(columnName)).name(columnName))
           convertedColumnNames.add(columnName)
+        } else if (!CommonUtils.isValidTFRecordType(dataFrameSchema(columnName).dataType)) {
```

Comment:
why?

Reply:
adding a safety check, TFRecord only supports primitive and array and arrayofarray types